crates/sui/src/client_commands.rs

```diff
@@ -1270,10 +1270,14 @@ impl Display for SuiClientCommandResult {
             SuiClientCommandResult::PayAllSui(cert, effects) => {
                 write!(writer, "{}", write_cert_and_effects(cert, effects)?)?;
             }
-            SuiClientCommandResult::Addresses(addresses) => {
+            SuiClientCommandResult::Addresses(addresses, active_address) => {
                 writeln!(writer, "Showing {} results.", addresses.len())?;
                 for address in addresses {
-                    writeln!(writer, "{}", address)?;
+                    if active_address.is_some() && active_address.unwrap() == *address {
+                        writeln!(writer, "{} <=", address)?;
+                    } else {
+                        writeln!(writer, "{}", address)?;
+                    }
```

Comment:
We could probably simplify this just a bit by doing the following:
```suggestion
                    if active_address == Some(*address) {
                        writeln!(writer, "{} <=", address)?;
                    } else {
                        writeln!(writer, "{}", address)?;
                    }
```

Reply:
Thank you @bmwill. I fixed and push a new commit applying your comment :)

9cef10603